tests/integration/rust/src/append.rs

```diff
@@ -33,14 +29,14 @@ struct TestFixture {
 }
 
 impl TestFixture {
-    async fn write_data_with_icelake(&mut self) {
-        let mut table = create_icelake_table(&self.args).await;
+    async fn write_data_with_icelake(&mut self, table_root: &str, csv_file: &str) {
```

Comment:
How about adding another called `append_partition_table` rather modifying current one. The benefit is that  we can clearly identify the error case on failure.

Reply:
We can split it into different test. But why not reuse this function. Seems it's function is general: write the data into the table.

Reply:
Yeah, I agree that we should reuse the code. You can refer to https://github.com/icelake-io/icelake/blob/26b7a0c99c690b1e8ed57a5fc5a29c8ef0caacb3/tests/integration/rust/src/tests.rs#L9 for adding another test case.